eval/src/apps/analyze_onnx_model/analyze_onnx_model.cpp

```diff
@@ -229,30 +260,34 @@ int probe_types() {
     if (!JsonFormat::decode(std_in, params)) {
         throw MyError{"invalid json"};
     }
+    MemoryUsage vm_before = extract_memory_usage();
     Slime result;
     auto &root = result.setObject();
     auto &types = root.setObject("outputs");
     Onnx model(params["model"].asString().make_string(), Onnx::Optimize::DISABLE);
     Onnx::WirePlanner planner;
-    for (size_t i = 0; i < model.inputs().size(); ++i) {
-        auto spec = params["inputs"][model.inputs()[i].name].asString().make_string();
+    for (const auto & i : model.inputs()) {
```

Comment:
consider better name for i, like model_input

Reply:
Fixed